waku/v2/node/waku_node.nim

```diff
@@ -21,6 +21,7 @@ import
   libp2p/protocols/connectivity/autonat/service,
   libp2p/nameresolving/nameresolver,
   libp2p/builders,
+  libp2p/protocols/rendezvous,
```

Comment:
Very much a nitpick: while `builders` above clearly doesn't follow this practice, we tend to sort imports more or less alphabetically (with the exception of common imports (`std` `chronos` `metrics` etc.) which are usually top-level). Not a strict rule, but would be great for this import to fit in with the other `libp2p/protocols` above :)